src/components/Login.tsx

```diff
@@ -95,24 +98,23 @@ const Login = ({ isPrimary, loginWalletOptions }: LoginProps): JSX.Element => {
 
   return (
     <div className="Login__block">
-      {!userId ? (
+      {!userId && !registrationPopoverVisible ? (
+        <LoginButton
```

Comment:
LoginButton and RegistrionModal are both popovers, so decided just to separate them into two popovers instead of passing the login button as one popover as a child of another popover.  it just simplified the logic, especially as the registration modal logic got more complex.